nestkernel/nest_time.cpp

```diff
@@ -162,7 +165,7 @@ Time::fromstamp( Time::ms_stamp t )
   // intended ones.
   tic_t n = static_cast< tic_t >( t.t * Range::TICS_PER_MS );
   n -= ( n % Range::TICS_PER_STEP );
-  long s = n / Range::TICS_PER_STEP;
+  long s = n * Range::TICS_PER_STEP_INV;
   double ms = s * Range::MS_PER_STEP;
```

Comment:
I agree with Alex.  To convert milliseconds into tics to step boundary, something like this
`ceil( ms * STEPS_PER_MS ) * TICS_PER_STEP`
will do the same, I believe. Also preserving infinity is an issue here. But however, I think this goes beyond this PR and would be a refactoring task.